.gitignore

```diff
@@ -108,4 +108,4 @@ ENV/
 *.~
 
 # idea
-.idea
\ No newline at end of file
+.idea
```

Comment:
This change is developer environment-specific so belongs in your `~/.config/git/ignore` rather than the project `.gitignore`. (Local `.gitignore` location varies a bit based on your configuration, yours might be `~/.gitignore` or something.)

Reply:
I am with @scanny on this one. Developers should have local gitignores that block those files from being committed.

Reply:
That being said, it's not a change, the file just has a newline added. The gitignore already had that in it.